app/views/layouts/_social_media.html.erb

```diff
@@ -1,11 +1,9 @@
-<% SiteConfig.social_media_handles.keys.each do |social_media_type| %>
-  <% if SiteConfig.social_media_handles[social_media_type].present? %>
-    <a
-      href="<%= "https://#{social_media_type}.com/#{SiteConfig.social_media_handles[social_media_type]}" %>"
-      target="_blank"
-      class="crayons-link crayons-link--secondary mx-2"
-      rel="noopener">
-      <%= inline_svg_tag("#{social_media_type}.svg", aria: true, title: social_media_type.capitalize, class: "crayons-icon social-media-svg") %>
-    </a>
-  <% end %>
+<% SiteConfig.social_media_handles.compact.each do |social_media_type, handle| %>
```

Comment:
1. `Hash#compact` removes KV-pairs where the value is `nil` from a hash, so we don't need the presence check.
    ```ruby
    SiteConfig.social_media_handles
    {
       "facebook" => nil,
         "github" => nil,
      "instagram" => nil,
         "twitch" => nil,
        "twitter" => "sloan"
    }
    SiteConfig.social_media_handles.compact
    {
      "twitter" => "sloan"
    }
    ```
2. Instead of iterating over `keys`, iterate over the hash and destructure it into type and handle to simplify the block.

Reply:
can they be empty string?

Reply:
I guess they theoretically could, so maybe `reject(&:blank?)` or `select(&:present?)` instead of `compact`.